File: Looping/Loop inside loop/Other Lang/C/LoopInLoop3.c
#include <stdio.h>

void main() {
	int num;
	for (int vert = 1; vert <= 3; vert++) {
		//Nested Loops Gave Me Anxiety...
		num = vert;
		for (int horz = 1; horz <= 7; horz++) {
			printf("%d ", num);
			num += 2;
		}
		printf("\n");
	}
}

File: Looping/Loop inside loop/Other Lang/Python/LoopInLoop3.py
def main():
	num = None
	for vert in range(1,4):
		num = vert
		# Nested Loops gave me anxiety, especially here...
		for horz in range(1,8):
			print(''.join([str(num), " "]), end="")
			num += 2
		print("");

main()
